treeherder/webapp/api/performance_data.py

```diff
@@ -1337,6 +1348,223 @@ def _get_signatures_map(signatures, grouped_values, option_collection_map):
+            "is_new_better": is_new_better,
+            "is_regression": is_regression,
+            "is_improvement": is_improvement,
+            "more_runs_are_needed": more_runs_are_needed,
+            "direction_of_change": direction,  # 'neutral', 'better', or 'worse'
+            "dke_isj_plot": {
+                "bandwidth": "Improved Sheather-Jones",
+                "kernel": "Gaussian",
+                "title": "Distribution Comparison Kernel Density Estimate (KDE)",
+                "xlabel": "Value",
+                "ylabel": "Density",
+                "summary_text": summary_text,
+                "base_kde": {
+                    "median": base_median,
+                    "sample_count": len(base),
+                    "kde_x": kde_x_base,
+                    "kde_y": kde_y_base,
+                },
+                "new_kde_plot": {
+                    "median": new_median,
+                    "sample_count": len(new),
+                    "kde_x": kde_x_new,
+                    "kde_y": kde_y_new,
+                },
+            },
+            "silverman_warnings": warning_msgs,
+            "silverman_kde": {
+                "bandwidth": "Silverman",
+                "base_mode_count": base_mode_count,
+                "base_peak_locs": base_peak_locs,
+                "base_prom": base_prom,
+                "new_mode_count": new_mode_count,
+                "new_peak_locs": new_peak_locs,
+                "new_prom": new_prom,
+                "mode_comments": [
+                    f"Estimated modes (Base): {base_mode_count} (location: {base_peak_locs}, prominence: {base_prom})",
+                    f"Estimated modes (New): {new_mode_count} (location: {new_peak_locs}, prominence: {new_prom})",
+                ],
+                "warnings": warning_msgs,
+                "mode_summary": mode_summary,
+                "median_shift_summary": median_shift_summary,
+                "ci_low": ci_low,
+                "ci_high": ci_high,
+                "shift": shift,
+                "shift_summary": ci_intepretation,
+                "is_regression": is_regression,
+                "is_improvement": is_improvement,
+                "ci_warning": ci_warning,
+            },
+            "cles": cles,
+            "cles_direction": direction,
+            "cles_interpretation": {
+                "cles": cles,
+                "cles_direction": cles_direction,
+                "cles_interpretation": interpretation,
+                "effect_size": effect_size,
+                "cles_explanation": cles_explanation,
+                "cles_comments": [
+                    f"**Mann-Whitney U Common Language Effect Size**: {cles_direction}",
+                    f"**p-value**: {mann_pvalue:.3f}, {'not' if mann_pvalue > pvalue_threshold else ''} significant",
+                    f"**Cliff's Delta**: {delta:.2f} → {interpretation}",
+                ],
+                "is_significant": is_significant,
```

Comment:
Will this value be displayed on the user interface?

Reply:
I think so?